<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lightPollutionWidget</class>
 <widget class="QWidget" name="lightPollutionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>34</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>1</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QComboBox" name="modeCB">
     <item>
      <property name="text">
       <string>automatic from locations database</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>manual</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>manual from SQM</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QSlider" name="manualSlider">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimum">
      <number>-999999999</number>
     </property>
     <property name="maximum">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="fromSQMmag_SB">
     <property name="decimals">
      <number>2</number>
     </property>
     <property name="maximum">
      <double>0.990000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="LogarithmicSpinBox" name="fromSQMcdm2_logSB"/>
   </item>
   <item>
    <widget class="QComboBox" name="unitCB">
     <property name="currentText">
      <string>cd/m²</string>
     </property>
     <item>
      <property name="text">
       <string>cd/m²</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>mcd/m²</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>μcd/m²</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>mag/arcsec²</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LogarithmicSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>LogarithmicSpinBox.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
